### ISSUE TITLE
ui: give more feedback indicating that the save button on the reporting form was successful

### ISSUE THREAD

**Issue by giskarda** [cockroachdb/cockroach#8077]
Hi,

By visiting : http://localhost:8080/#/help-us/reporting

After filling the form, clicking on the submit button nothing happens.

The button apparently doesn't include any 'action'.

```
[~/src/github.com/cockroachdb/cockroach] ➔  cat ui/app/containers/helpus.tsx | grep button
          <button className="right">Submit</button>
[~/src/github.com/cockroachdb/cockroach] ➔
```

```
[~/src/github.com/cockroachdb/cockroach] ➔  ./cockroach start
build:     beta-20160721-171-gfc3ea0e @ 2016/07/27 21:25:56 (go1.6.3)
admin:     http://localhost:8080
sql:       postgresql://root@localhost:26257?sslmode=disable
logs:      cockroach-data/logs
store[0]:  path=cockroach-data
```

**Comment by maxlang** [comment]
Thanks for the report @giskarda 

Currently the data isn't sync'd with cockroach (though it will be in an upcoming version) but it should be saved on the backend.

Try running  `SELECT * FROM system.ui;` to see that it's saved on the backend.

That said, it would help to give more feedback indicating that the save was successful.

As far as how this is implemented in the code, the `form` element has an `onSubmit` field in `helpus.tsx`.

**Comment by giskarda** [comment]
@maxlang you are right;
```
root=> SELECT * FROM system.ui;
-[ RECORD 1 ]-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
key         | helpus
value       | \x7b22656d61696c223a2272736574746940676d61696c2e636f6d222c226f7074696e223a747275652c226469736d6973736564223a312c2266697273746e616d65223a22526963636172646f222c226c6173746e616d65223a225365747469222c22636f6d70616e79223a22222c2275706461746573223a6e756c6c7d
lastUpdated | 2016-07-27 21:38:11.982716

root=>
```

and yeah, I should have read this as well.

 // TODO: add "saving..." text and show/hide the required text

Feel free to close this if you think its the right thing to do.

**Comment by mrtracy** [comment]
This has been addressed in #12982, which is a new design for this page.  This issue should be closed when that work is completed.

**Comment by dianasaur323** [comment]
@giskarda we are actually doing a pretty big revamp of this page, so no longer an issue at this point. Thanks for your feedback!!
